include/boost/astronomy/coordinate/base_equatorial_frame.hpp

```diff
@@ -2,245 +2,243 @@
 #define BOOST_ASTRONOMY_COORDINATE_BASE_EQUATORIAL_FRAME_HPP
 
 #include <type_traits>
-#include <string>
-
-#include <boost/geometry/core/cs.hpp>
-#include <boost/geometry/geometries/point.hpp>
-#include <boost/geometry/algorithms/transform.hpp>
-#include <boost/geometry/algorithms/equals.hpp>
-#include <boost/lexical_cast.hpp>
-
-#include <boost/astronomy/detail/is_base_template_of.hpp>
 #include <boost/astronomy/coordinate/base_frame.hpp>
 #include <boost/astronomy/coordinate/spherical_representation.hpp>
 #include <boost/astronomy/coordinate/spherical_coslat_differential.hpp>
 
-namespace boost
+namespace boost { namespace astronomy { namespace coordinate {
+
+namespace bu = boost::units;
+namespace bg = boost::geometry;
+
+template
+<
+    typename Representation, typename Differential
+>
+struct base_equatorial_frame : public base_frame
+    <Representation, Differential>
```

Comment:
put this in 19th line with `base_frame` if it does not exceed the line length limit